config/middlewares.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [
+  'strapi::errors',
+  'strapi::security',
+  'strapi::cors',
+  'strapi::poweredBy',
+  'strapi::logger',
+  'strapi::query',
+  'strapi::body',
+  'strapi::session',
+  'strapi::favicon',
+  'strapi::public',
+]
```

Comment:
Is this Strapi v4 related or should it be in this PR?

Reply:
Yes